scripts/scripts/enrich_sentences.py

```
@@ -0,0 +1,223 @@
+import argparse
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)